OgreMain/include/Compositor/OgreCompositorManager2.h

```diff
@@ -52,6 +52,17 @@ namespace Ogre
     typedef vector<TextureGpu*>::type TextureGpuVec;
     typedef vector<UavBufferPacked*>::type UavBufferPackedVec;
 
+    namespace ExecutionFlags
+    {
+        // Bit mask
+        // | -Z | +Z | -Y | +Y | -X | +X | n/a | No slice | 24-bits reserved
+        const uint32 FIRST_SLICE_EXECUTION_FLAG = 0x4000000;
+        const uint32 NO_SLICE_EXECUTION_FLAG    = 0x1000000;
+        const uint32 RESERVED_EXECUTION_FLAGS   = 0x0ffffff;
+
+        const uint32 DEFAULT_EXECUTION_FLAGS    = NO_SLICE_EXECUTION_FLAG | RESERVED_EXECUTION_FLAGS;
+    }
```

Comment:
- I declared these const in this file because below there are some references to them. What do you prefer @darksylinc? Leave them where they are vs move them to a new/different file?
- @darksylinc, what do you think about using the 25th bit to skip the mipmap generation (in case of DPM)? Doing so, the user wouldn't need to deal with execution bits (as a parameter of the `CubemapProbe::initWorkspace` method and in the cubemap probe `.compositor` file)

Reply:
> * @darksylinc, what do you think about using the 25th bit to skip the mipmap generation (in case of DPM)? Doing so, the user wouldn't need to deal with execution bits (as a parameter of the `CubemapProbe::initWorkspace` method and in the cubemap probe `.compositor` file)

There's no need anymore for end users to deal with `execution_mask` in `.compositor` files, appropriate `ExecutionFlags` are set automatically. This is the new bit mask:
```
// Bit mask
// | IBL | -Z | +Z | -Y | +Y | -X | +X | No TS | 24-bits reserved
const uint32 TS_IBL_EXECUTION_FLAG        = 0x80000000;
const uint32 TS_FIRST_FACE_EXECUTION_FLAG = 0x02000000;
const uint32 TS_NONE                      = 0x01000000;
const uint32 RESERVED_EXECUTION_FLAGS     = 0x00FFFFFF;

const uint32 DEFAULT_EXECUTION_FLAGS      = TS_NONE | RESERVED_EXECUTION_FLAGS;
```